common/theme.h

```diff
@@ -32,3 +33,5 @@ bool colorFromSetting(config_setting_t *rgba, color_t *col);
 void themeStartup(ThemePreset preset);
 
 theme_t themeCurrent;
```

Comment:
Copy these into theme.c and add extern prefix to these in the .h.

Reply:
done